### PR TITLE
influxdb: Fix documentation

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
lib/ansible/module_utils/influxdb.py
lib/ansible/modules/database/influxdb/influxdb_database.py
lib/ansible/modules/database/influxdb/influxdb_query.py
lib/ansible/modules/database/influxdb/influxdb_retention_policy.py
lib/ansible/modules/database/influxdb/influxdb_user.py
lib/ansible/modules/database/influxdb/influxdb_write.py
test/sanity/ignore.txt

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @felixfontein @Andersson007

File: lib/ansible/modules/database/influxdb/influxdb_database.py [hunk 1]
```diff
@@ -29,11 +29,13 @@
         description:
             - Name of the database.
         required: true
+        type: str
     state:
         description:
             - Determines if the database should be created or destroyed.
         choices: [ present, absent ]
```
Comment (Andersson007) [inline lib/ansible/modules/database/influxdb/influxdb_database.py]:
```suggestion
        choices: [ absent, present ]
```

**Review by Andersson007 — CHANGES_REQUESTED** [review]
- about alphabetical order of list elements.
- ```DOCUMENTATION = '''``` -> ```DOCUMENTATION = r'''``` and the same for the EXAMPLES and RETURN blocks in some files.

**Review by resmo — APPROVED** [review]
shipit

**Pull request merged**